ekglib/main/main.py

```diff
@@ -5,13 +5,16 @@
 from rdflib import Graph, plugin
 
 from ..log import error, log_item
+from ..kgiri import kgiri_replace_iri_in_graph
 
 
 def load_rdf_file_into_graph(graph: rdflib.Graph, rdf_file: pathlib.Path):
     if not rdf_file.exists():
         error(f"File does not exist: {rdf_file}")
     with rdf_file.open() as f:
         graph.parse(source=f, format='turtle')  # TODO: Detect format so that we can support any RDF file
+        kgiri_replace_iri_in_graph(graph)
```

Comment:
Now this function does more that it says in the name. 
Preferably: push this call outside of the load function.
If that's not practical, rename the function to say it loads the file and replaces IRIs